.env.tpl

```diff
@@ -2,7 +2,8 @@
 
 # uncomment to try out deploying the w3up api under a custom domain.
 # the value should match a hosted zone configured in route53 that your aws account has access to.
-# HOSTED_ZONE=up.dag.haus
+# HOSTED_ZONE=up.web3.storage
+# HOSTED_ZONE_STORACHA=up.storacha.network
```

Comment:
Do we actually want to support multiple domains? I envisioned this as something very temporary. Note that we have forking code paths for the emails and Preact pages. I don't think we actually want something like that indefinitely, just long enough to cover the launch of Storacha. What's in this PR seemed to me like the easiest thing to tear out later. Making `HOSTED_ZONE` plural is a bunch of logic if we don't actually want to keep it for long.

Reply:
> Do we actually want to support multiple domains? I envisioned this as something very temporary.

I don't know for sure. In the past we've wanted to theme nft.storage emails, and we literally want this now. I don't see a lot of harm in retaining the logic. I find that temporary becomes permanent more often than I expect.

> Note that we have forking code paths for the emails and Preact pages. I don't think we actually want something like that indefinitely, just long enough to cover the launch of Storacha. What's in this PR seemed to me like the easiest thing to tear out later.

I think I also suggested the forking could instead just be a keyed map of config (or at least I meant to). It would be quicker to not tear this out later and just reconfigure...

> Making `HOSTED_ZONE` plural is a bunch of logic if we don't actually want to keep it for long.

I guess yes, but I feel like pulling this out vs a plural `HOSTED_ZONE` would be approximately the same effort.